docs/release-notes/highlights-1.3.0.asciidoc

```diff
@@ -0,0 +1,44 @@
+[[release-highlights-1.3.0]]
+== 1.3.0 release highlights
+
+[float]
+[id="{p}-130-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 1.3.0 of {n}. See <<release-notes-1.3.0>> for the full list of changes.
+
+[float]
+[id="{p}-130-official-helm-chart"]
+==== Official Helm chart
+
+Addressing one of the most common asks, ECK 1.3.0 introduces <<{p}-install-helm,an official Helm chart>> which allows to configure and deploy ECK. Apart from enabling new deployments, a migration path that enables moving from previous deployment method to a Helm chart is documented.
+
+[float]
+[id="{p}-130-volume-expansion-support"]
+==== Volume expansion support
+
+Elasticsearch Pods volumes can be now dynamically expanded. A single edit to the resource YAML manifest is all that's needed to grow the volumes. In addition to that, in many cases (including GCE-PD, AWS-EBS, Azure, and TopoLVM), it is a hot filesystem resize that doesn't require Pod restart, and it's active seconds after the change was made.
```

Comment:
```suggestion
Elasticsearch Pods volumes can be now dynamically expanded. A single edit to the resource YAML manifest is enough to grow the volumes. In addition to that, in many cases (including GCE-PD, AWS-EBS, Azure, and TopoLVM), it is a hot filesystem resize that doesn't require Pod restart, and it's active seconds after the change was made.
```

Reply:
```suggestion
Elasticsearch volumes can be now dynamically expanded. A single edit to the resource YAML manifest is enough to grow the volumes. In many cases (dependent on the underlying storage provider), this can be done without even restarting the pod and so expansions complete in seconds.
```